sxs/waveforms/alignment.py

```diff
@@ -408,6 +427,14 @@ def align4d(
     nprocs: int, optional
         Number of cpus to use.  Default is maximum number.  If -1 is provided,
         then no multiprocessing is performed.
+    max_nfev: None or int, optional
+        Parameter for scipy.optimize.least_squares.
+        Controls the maximum number of function evaluations used. 
+        If None (default), the value is 100 * number of parameters
```

Comment:
Docs claim default value is None, but on line 378 above, the default is specified as 50000